node_modules/oae-authentication/emailTemplates/reset.txt.jst

```diff
@@ -0,0 +1,21 @@
+<%
+    var greeting = util.i18n.translate('__MSG__ACTIVITY_EMAIL_GREETING__', {'displayName': util.html.encodeForHTML(user.displayName)});
+    greeting = util.html.toText(greeting, false);
+%>
+
+<%= greeting %>
+
+Please try to reset your password by visiting the following URL:<br />
```

Comment:
`<br />` can be removed as it doesn't make much sense in the context of a text email.